inet/src/inet/common/queue/DropTailQueue.cc

```diff
@@ -97,6 +101,36 @@ cMessage *DropTailQueue::enqueue(cMessage *msg)
     cPacket* pkt = check_and_cast<cPacket*>(msg);
     queue.insert(pkt);
 
+    if(logPacketEvents){
```

Comment:
The following code is repeated few times. Can you make it a utility function?

Reply:
The logging inside these if conditions are different depending upon the place of logging. I'll however try to see if if I can make this a utility function across the code.